linux_os/guide/services/ssh/ssh_server/sshd_disable_allow_tcp_forwarding/tests/comment.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp
```

Comment:
please remove these #profiles lines from all tests. They should not be there if the test is not specific for certain profile. In this case I think it is not.